packages/aurora-feed/FeedEditor.js

```diff
@@ -9,6 +9,10 @@ const CardAtBottom = Card.extend`
   margin-bottom: 0;
   border-top: 1px solid
     ${props => (props.theme ? props.theme.colors.border : "black")};
+  .public-DraftEditorPlaceholder-inner {
```

Comment:
So this is the proper place for the styled components. Ideally, it should have been one layer lower down, at the actual Editor level, but since this is a class we do not have access to (comes from draft-js) library, we have done our best to place the css as close to the component as possible. This is the idea with styled components. We want the css to belong to specific components instead of being global and thus affecting random things we don't want it to affect.

Reply:
Yup! And this looks good. We can't always get everything, but this is as close to the metal as we can get. A+ work team!